docs/source/usage.rst

```diff
@@ -421,7 +422,8 @@ The following example sets ``abc123`` as the default value when ``DB_PASSWORD``
 Decoding strings with interpolations
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can automatically convert a string to its corresponding type (e.g., bool, int, float, dict, list) using ``oc.decode``.
+You can automatically convert a string to its corresponding type (e.g., bool, int, float, dict, list) using ``oc.decode``
```

Comment:
It's not automatic, you are using the oc.decode resolver :).

Reply:
It's automatic in the sense that you don't have to specify the target type.
Anyway, this was rephrased when I added the resolution of interpolations => resolving.